src/modules/Topics/Topics.tsx

```diff
@@ -108,19 +104,21 @@ const Topics: React.FC = () => {
     <Card className='kafka-ui-m-full-height' data-ouia-page-id='tableTopics'>
       <KafkaTopics
         topics={topicsList}
-        getUrlFortopic={(row) => `${basename}/topics/${row.topic_name}`}
-        onDelete={(row) => onDelete(row.topic_name)}
-        onEdit={(row) => onEdit(row.topic_name)}
-        onSearchTopic={(value) => {
+        getUrlFortopic={(row: { name: any }) =>
+          `${basename}/topics/${row.name}`
+        }
+        onDelete={(row: { name: string }) => onDelete(row.name)}
+        onEdit={(row: { name: string | undefined }) => onEdit(row.name)}
+        onSearchTopic={(value: string) => {
           topicsChips.clear();
           topicsChips.toggle(value);
         }}
         onClearAllFilters={topicsChips.clear}
         topicName={topicsChips.chips}
         onRemoveTopicChip={topicsChips.clear}
         onRemoveTopicChips={topicsChips.clear}
-        onTopicLinkClick={(row) => {
-          row.topic_name;
+        onTopicLinkClick={(row: { name: any }) => {
```

Comment:
same here

Reply:
I guess It got added when I was manually importing the UI components. fixed it